samples/app-anonymous-users/csharp/FacebookDocumentation/README.md

```diff
@@ -0,0 +1,29 @@
+# Create and facebook app.
```

Comment:
Rename folder name as FacebookAuth

Reply:
Link this readme with main readme in your sample

Reply:
Updated folder name and main readme.